SCClassLibrary/Common/Files/SoundFile.sc

```diff
@@ -70,6 +71,38 @@ SoundFile {
 		^res
 	}
 
+    *writeArray {|pathName, array, headerFormat, sampleRate, sampleFormat|
```

Comment:
actually, you could put the array first and then the pathName. Then one could still add an automatic pathName later (like in Recorder.makePath), and you could write `SoundFile.writeArray(array);` Also `pathName, headerFormat, sampleFormat` always come in a row.